tests/lib/rules/no-fallthrough.js

```diff
@@ -109,8 +130,116 @@ ruleTester.run("no-fallthrough", rule, {
         {
             code: "switch (a) {\n case 1: ; break; \n case 3: }",
             options: [{ allowEmptyCase: false }]
+        },
+        `
+switch (foo) {
+    case 0:
+        a();
+}
+switch (bar) {
+    case 1:
+        b();
+}
+            `,
+        {
+            code:
+        `
+switch (foo) {
+    case 0:
+        a();
+        break;
+        // falls through
+}
+switch (bar) {
+    case 1:
+        b();
+}
+            `,
+            options: [{ reportUnusedFallthroughComment: true }]
+        },
+        {
+            code:
+        `
+switch (foo) {
+    case 0:
+        a();
+        break;
+        /* falls through */
+}
+switch (bar) {
+    case 1:
+        b();
+}
+            `,
+            options: [{ reportUnusedFallthroughComment: true }]
+        },
+        {
+            code: `
+switch(foo){
+    case 1:
+        doSomething();
+        break;
+    // just a comment
+    case 2: doSomething();
+}
+          `,
+            options: [{ reportUnusedFallthroughComment: true }]
+
+        },
```

Comment:
Can you also add a valid test with a fallthrough comment, for example:

```js
        {
            code: `
switch(foo){
    case 1:
        doSomething();
        // falls through
    case 2: doSomething();
}
          `,
            options: [{ reportUnusedFallthroughComment: true }]

        },
```

Reply:
makes sense! added